src/main/java/de/craften/ui/swingmaterial/fonts/MaterialIcons.java

```diff
@@ -0,0 +1,976 @@
+package de.craften.ui.swingmaterial.fonts;
+
+import java.awt.Font;
+import java.awt.FontFormatException;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Helper class for getting access to the Material Icon set as a TrueType font.
+ * Both the Font and the known chars for each icon are available from here.
+ * @author DragShot
+ */
+public class MaterialIcons {
+    public static final char _3D_ROTATION = '\uE84D';
```

Comment:
:smile: 
When I'd search for this icon, I'd probably type "rotation 3d" and still find "rotation_3d" as well as "_rotation_3d". I just don't like the leading underscore.